include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -317,7 +313,9 @@ namespace picongpu
                             numJobs = 1;
                         }
                         // allocate memory for all amplitudes for temporal data collection
-                        tmp_result = new Amplitude[elements_amplitude()];
+
+                        // ACCUMULATOR! Should be in double precision
+                        tmp_result = new Amplitude<>[elements_amplitude()];
```

Comment:
Not related to this PR, but perhaps as a separate refactoring these raw pointers want to become `unique_ptr`s ?

Reply:
Nevermids, those are arrays. Maybe `std::vector`s then?

Reply:
I think this should be possible, since the data in `Radiation.hpp` resides on the host.

Reply:
See below.